README.md

```diff
@@ -1 +1,49 @@
-# Ruby-Catalog
\ No newline at end of file
+# Ruby Catalog
+An app for your personal catalog that can shows and stores your favourite books , music albums , games.
```

Comment:
- [ ] Please correct the grammatical error here. Additionally, you can  run your readme file through Grammarly to check for errors 🙂